### ISSUE TITLE
AltGr key broken under Linux

### ISSUE THREAD

**Issue by nowhereman** [MikeRatcliffe/Acebug#32]
It inserts space when we press it.
But with French keyboard layout, this key is used in combination with other keys to write `{` or `[` characters and many others.

Patch :  You should replace that line [startup.js#L201](https://github.com/MikeRatcliffe/Acebug/blob/master/chrome/content/ace++/startup.js#L201) with this  one :

``` javascript
if (navigator.appVersion.indexOf("Win")!=-1 && keyCode==0 && e.type == "keypress")
```

**Comment by nowhereman** [comment]
Fixed in Acebug 1.9, available [here](https://addons.mozilla.org/en-US/firefox/addon/acebug/versions).

**Comment by nightwing** [comment]
i didn't know about special use of AltGr key on international keyboards, and somewhy had en-us international layout on one of my machines, 
thanks for reporting this
